resources/views/index.blade.php

```diff
@@ -66,6 +66,10 @@
         })
 
         window.ui = ui
+
+        ui.initOAuth({
+            usePkceWithAuthorizationCodeGrant: "{!! (bool)config('l5-swagger.defaults.ui.authorization.use_pkce_with_authorization_code_grant') !!}"
+        })
```

Comment:
Should we wrap this with `if`:
```suggestion
       @if((bool)config('l5-swagger.defaults.ui.authorization.use_pkce_with_authorization_code_grant'))
        ui.initOAuth({
            usePkceWithAuthorizationCodeGrant: "{!! (bool)config('l5-swagger.defaults.ui.authorization.use_pkce_with_authorization_code_grant') !!}"
        })
        @endif
```
Or this call `ui.initOAuth` will not change default behaviour if we are not using oAuth?

Reply:
Good idea, I suggest checking that at least one of the security schemes ```type``` is ```oauth2```. Only with this condition we add ```ui.initOauth```. Besides that, I moved option ```use_pkce_with_authorization_code_grant``` to new ```oauth2``` array element, I think in such way it is more readable.
Please, see details in my latest commit.